packages/editor/src/components/block-navigation/index.js

```diff
@@ -24,7 +25,10 @@ function BlockNavigationList( {
 	showNestedBlocks,
 } ) {
 	return (
-		<ul className="editor-block-navigation__list">
+		<ul
+			aria-label={ __( 'Block Navigation' ) }
```

Comment:
Oh, @youknowriad pinged me and pointed out this shouldn't be here; we should move this into the block as content (an off-screen header I guess).

I used it with NVDA this way and it only read the label when I used the keyboard shortcut, not when I activated the dropdown using the toggle button. The toggle button provides context as to what this is so it seemed like noise to add it to the content.

But if it's not too annoying to read it out again after the button announcing the content we can move it in, sure 😄